### ISSUE TITLE
IndexError in skip_first_whitespace

### ISSUE THREAD

**Issue by izquierdo** [Kozea/WeasyPrint#1011]
Hi, I've hit an `IndexError: list index out of range` crash that seems very similar to https://github.com/Kozea/WeasyPrint/issues/783 and https://github.com/Kozea/WeasyPrint/issues/1009. This happens in WeasyPrint 50 with the following snippet:

```html
<html>
<head>
<style>
@page {
    margin: 1cm;
}
</style>
</head>
<body>
<div style="max-width: 45%">
<h2><wbr>トトトト　　　　　　　　　　トトトト　　トトトトトトトト　ト　<wbr>ト</h2>
</div>
</body>
</html>
```

Note that the whitespace within the `<h2>` is made of `'\u3000'` characters.

**Comment by Tontyna** [comment]
Cannot reproduce the error -- probably due to differences in my font. But I wonder whether (hope that) it's fixed by #956.

@izquierdo, could you test your snippet with the current master branch?

**Comment by izquierdo** [comment]
@Tontyna tested with the master branch and confirmed the issue does not occur. Thank you!
